src/Provider/AbstractProvider.php

```diff
@@ -446,6 +446,7 @@ protected function getAccessTokenBody(array $params)
     protected function getAccessTokenOptions(array $params)
     {
         $options = [];
+        $options['headers'] = ['content-type', 'application/x-www-form-urlencoded'];
```

Comment:
I like one line better! Sometimes "coding style" opinions show up ;) 

If you are in support of one line here, so am I. Commit updated.